int16 bb_number
handsnet_tftrt_yolo/BB[] bb_array